yumemi-ios-training/View/WeatherViewController.swift

```diff
@@ -110,7 +117,7 @@ class WeatherViewController: UIViewController {
         )
     }
     
-    private func reloadWeather() {
+    @objc func reloadWeather() {
```

Comment:
[nits] 外部に公開する必要はないかと思います。
```suggestion
    @objc private func reloadWeather() {
```